Assets/Scripts/CPUConfig.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "CPUConfig", menuName = "Game/CPUConfig")]
+public class CPUConfig : ScriptableObject
+{
+    [System.Serializable]
+    public class ModeSettings
+    {
+        public string modeName;     // モード名 (例: Easy, Normal, Hard)
+        public float moveSpeed;     // 横移動速度
+        public float rotationSpeed; // 回転速度
+    }
```

Comment:
変更できないようにしよう。モード名が3つで固定なら、Enumで定義してあげると安全かもしれない（置く場所は任せます）。
```suggestion
    public class ModeSettings
    {
        [SerializeField] private CPUMode _mode;        // モード (例: Easy, Normal, Hard)
        [SerializeField] private float _moveSpeed;     // 横移動速度
        [SerializeField] private float _rotationSpeed; // 回転速度

        public CPUMode Mode => _mode;
        public float MoveSpeed => _moveSpeed;
        public float RotationSpeed => _rotationSpeed;
    }

    public enum CPUMode
    {
        Easy,
        Normal,
        Hard,
    }
```